helm/kiam-app/templates/agent-np.yaml

```diff
@@ -1,5 +1,5 @@
-kind: NetworkPolicy
-apiVersion: networking.k8s.io/v1
+apiVersion: cilium.io/v2
```

Comment:
I would rather choose to use 

```
{{ if .Values.ciliumNetworkPolicy.enabled }}
```

and adding a value

```
ciliumNetworkPolicy:
  enabled: false
```

to check if we can apply it. 

This would help us in two ways.

1.) If we need to draft a release for 18.x.x for kiam we would avoid changing everything back 
2.) This will allow us to only apply cilium network policy in case cluster-operator set the cluster value ciliumNetworkPolicy to `true`

Reply:
The app has the LocalRedirectPolicy and also disabled iptables completely so it is a bit more complex.

If we want to keep compatibility with both CNI I will have to add a few more things.

Reply:
I see, it's tricky. Ok let's do a major release for kiam, don't wanna block you here.